### DISCUSSION TITLE
有没有可能打包多个网站到一个app里面？

### DISCUSSION THREAD

**Discussion by aliuq** [tw93/Pake#349] [Q&A]
每个网站一个app太多了，有没有可能打包多个网站到一个app里面？

**Comment by Tlntin** [comment]
比如自己搭建一个导航页，然后吧各个app超链接进去。
或者更简单一点，下载谷歌浏览器，把链接加入到书签

**Reply by aliuq** [reply]
这样可能是要简单些了
